src/tui/mode/helix.zig

```diff
@@ -60,20 +60,16 @@ const cmds_ = struct {
     }
     pub const wq_meta: Meta = .{ .description = "wq (write/save file and quit)" };
 
+    pub fn @"x!"(_: *void, _: Ctx) Result {
+        try cmd("save_file", command.fmt(.{ "then", .{ "quit_without_saving", .{} } }));
+    }
+    pub const @"x!_meta": Meta = .{ .description = "x! (write/save file and close forcefully, ignoring other unsaved changes)" };
```

Comment:
This can be "x! (write/save file and exit ignoring other unsaved changes)".

"close forcefully" may cause confusion with `force_terminate`, which also sets an error exit code.

Reply:
Correct, verified and we want to quit, not terminate. Thank you.